kornia/geometry/warp/homography_warper.py

```diff
@@ -6,16 +6,92 @@
 
 from kornia.utils import create_meshgrid
 from kornia.geometry.linalg import transform_points
+from kornia.testing import check_is_tensor
 
 
 __all__ = [
     "HomographyWarper",
     "homography_warp",
+    "warp_grid",
     "normalize_homography",
     "normal_transform_pixel",
 ]
 
 
+def warp_grid(grid: torch.Tensor, src_homo_dst: torch.Tensor) -> torch.Tensor:
```

Comment:
probably it's out of the scope o this PR but at as comment for future refactoring whoever takes it, this function doesn't compute the warping for a grid it just create it as `torch.nn.functional.affine_grid` already does. We should explore to use `affine_grid` for the homography case since e.g. in 2d affine transforms we saw that the speed for this simple function using torch cpp (affine_grid) improves by far the speed performnance.